nservicebus/sagas/index.md

```diff
@@ -71,9 +71,9 @@ To ensure messages are not discarded when they arrive out of order:
 
 If multiple message types can start a saga, it's necessary to ensure that saga behavior will remain correct for all possible combinations of order in which those messages are received.
 
-Assume that message `A` should be processed before message `B`. Often in such a situation the saga will persist some data from message `A` to be read when the saga processes message `B`. However, if message `B` is processed first and starts the saga, then the data from message `A` has not been persisted yet.
+In the previous example, the `StartOrder` message contains both `OrderId` and `CustomerId` but the `CompleteOrder` message only contains the `OrderId`. The handler for `CompleteOrder` requires `CustomerId` to complete the order saga (e.g. initiate a shipping process). When `StartOrder` is processed first, the saga persists both `OrderId` and `CustomerId` in the saga data and when `CompleteOrder` message arrives second, it retrieves `CustomerId` from saga data.
```

Comment:
Some tweaks:

In the previous example, the `StartOrder` message contains both an `OrderId` and a `CustomerId` but the `CompleteOrder` message contains only an `OrderId`. The handler for `CompleteOrder` requires `CustomerId` to complete the order saga (i.e. initiate a shipping process). If a `StartOrder` message is processed first, the saga persists both the `OrderId` and the `CustomerId` in the saga data and when a corresponding `CompleteOrder` message arrives later, it retrieves the `CustomerId` from the saga data.